db/seeders/users_seeder.rb

```diff
@@ -1,29 +1,41 @@
-class UserSeeder
-  attr_accessor :organization, :user_params
+class UsersSeeder
+  attr_accessor :pdx_org, :sf_org
 
-  def self.seed(user_params, organization = nil)
-    new(user_params, organization).seed
+  def self.seed(pdx_org, sf_org)
+    new(pdx_org, sf_org).seed
   end
 
-  def initialize(user_params, organization = nil)
-    @user_params = user_params
-    @organization = organization
+  def initialize(pdx_org, sf_org)
+    @pdx_org = pdx_org
+    @sf_org = sf_org
   end
 
   def seed
-    create_user
+    users.map {|user| create_user(user) }
   end
 
   private
 
-  def create_user
+  def users
+    [
+      { email: 'superadmin@example.com', organization_admin: false, super_admin: true},
+      { email: 'org_admin1@example.com', organization_admin: true, organization: pdx_org },
+      { email: 'org_admin2@example.com', organization_admin: true, organization: sf_org },
+      { email: 'user_1@example.com', organization_admin: false, organization: pdx_org },
+      { email: 'user_2@example.com', organization_admin: false, organization: sf_org },
+      { email: 'test@example.com', organization_admin: false, super_admin: true, organization: pdx_org },
+      { email: 'test2@example.com', organization_admin: true, organization: pdx_org }
+    ]
+  end
```

Comment:
This is a really minor thing, but I'll just mention it...

The `users` method might be assumed to return User objects, but instead returns user hashes. I would either rename it to `user_hashes`, or just move the array into the `seed` method and remove the `users` method altogether.